### PR TITLE
Add more Talk kinds and improve the matching of existing kinds

### PR DESCRIPTION
**Pull request by marcoroth**
We have a lot more different kinds of "talks" in our current dataset. This pull request adds a few more kinds to better represent our dataset:

Follow up on #334

### CODE REVIEW

File: app/models/talk.rb [hunk 1]
```diff
@@ -356,18 +358,33 @@ def suggestion_summary
...
+    when /.*(fishbowl:|fishbowl\ discussion:).*/i
+      :fishbowl
+    when /.*(podcast:|podcast\ recording:|live\ podcast:).*/i
+      :podcast
+    when /.*(q&a|q&a:|ama|q&a\ with|ruby\ committers\ vs\ the\ world|ruby\ committers\ and\ the\ world).*/i
+      :q_and_a
+    when /.*(discussion:|discussion).*/i
+      :discussion
+    when /.*(fireside\ chat:|fireside\ chat).*/i
+      :fireside_chat
```
Comment (adrienpoly) [inline app/models/talk.rb:392]: fireside_chat isn't it a kind of q_and_a?

same for panel, fishbowl, discussion aren't they a similar kind at the end but with different naming in conferences?

Comment (marcoroth) [reply app/models/talk.rb:392]: Yeah, they are close to each other, but I guess the formats have different nuances which make them unique.

Comment (marcoroth) [reply app/models/talk.rb:392]: But maybe we can group them somehow that makes sense?

Comment (adrienpoly) [reply app/models/talk.rb:392]: I wonder if someone would search specifically for a fishbowl ?

Comment (marcoroth) [reply app/models/talk.rb:388]: I combined them 👍🏼

**Pull request merged**